### PR TITLE
Add git command

### PR DESCRIPTION
**Pull request by Sispheor**
I tried to make it smaller as I could. I hope it's ok.

### CODE REVIEW

**Comment by scorphus** [comment]
Thanks again, @Sispheor! My comments in #46, regarding unnecessary `.gitignore` additions, three lines before a heading and squashing all commit into one, are valid here as well. Beside those, I have some comments that may be valid for #46 as well:

File: eg/examples/git.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+# git
+
+Tel Git who you are
```
Comment (scorphus) [inline eg/examples/git.md]: A small typo here, `s/Tel/Tell/`.

**Comment by scorphus** [comment]
I see a lot of trailing spaces. Can you please configure your editor to automatically strip them on save? You'll benefit from this as well :smiley: 

Also,  8b09868 is already requested at #46.

**Comment by scorphus** [comment]
@Sispheor I've also updated the branch `add_git_command` which is the result of the following recorded terminal session:

[asciicast](https://asciinema.org/a/5tqqjtaoo7pr7iotqm4ajqhml)

I hope this is helpful as well :wink:

**Comment by scorphus** [comment]
@Sispheor: I forgot to remove the trailing spaces, add those block of three lines after each section and fix that `s/Tel/Tell/` typo. I've done so, then `git commit --amend` and then `git push _peer/_Sispheor add_git_command -f` to have it updated. Please check.

**Comment by srsudar** [comment]
I'm confused! This one is closed? Has it been merged somewhere?

**Comment by Sispheor** [comment]
I don't know. The PR has been closed?

2015-11-17 16:59 GMT+01:00 Sam Sudar <notifications@github.com>:

> I'm confused! This one is closed? Has it been merged somewhere?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/srsudar/eg/pull/47#issuecomment-157413153>.
>



-- 
*Nicolas MARCQ*

**Comment by srsudar** [comment]
I'm seeing "Closed" as the icon at the top of the page. I should be able to still merge it in, but I was wondering if anyone knew why it says closed.

**Comment by scorphus** [comment]
@Sispheor: You can reopen both pull requests and changes made to their now updated branches will reflect automatically – :heart: GithHub

**Comment by Sispheor** [comment]
done

2015-11-17 17:32 GMT+01:00 Pablo Aguiar <notifications@github.com>:

> @Sispheor <https://github.com/Sispheor>: You can reopen both pull
> requests and changes made to their now updated branches will reflect
> automatically – [image: :heart:] GithHub
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/srsudar/eg/pull/47#issuecomment-157422492>.
>



-- 
*Nicolas MARCQ*

**Pull request closed without merging**